spec/controllers/people/picture_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+describe People::PictureController do
+  before(:each) do
+    sign_in auth_users(:user), scope: :auth_user
+  end
+
+  it 'should show default_avatar' do
+    get :show, params:{ id: bob.id }
+
+    default_avatar_md5 = '96643270fe346cc83165128dfcfccd4c'
+
+    expect(response.status).to eq(200)
+    expect(Digest::MD5.hexdigest(response.body)).to eq(default_avatar_md5)
+  end
+
+  it 'should update picture' do
+    process :update, method: :put, params: { id: bob.id, picture: fixture_file_upload('picture.png', 'image/png') }
+
+    path = json['data']['picture_path']
+
+    bob.reload
+
+    expect(response.status).to eq(200)
+    expect(bob['picture']).to eq('picture.png')
+    expect(path).to eq('/people/902541635/picture')
```

Comment:
Magic number, are you sure this will always be the same?
Fairly certain the fixtures don't have fixed ids.
Therefore you can just interpolate `bob.id`